script.js

```diff
@@ -0,0 +1,150 @@
+const toDoForm = document.querySelector(".toDoForm");
+const toDoInput = toDoForm.querySelector("input");
+const toDoList = document.querySelector(".toDoList");
+const finishedList = document.querySelector(".finishedList");
+const lenTodo = document.getElementById("todo-count");
+const lenFin = document.getElementById("fin-count");
+
+const TODODATA = "todoData"
+const FINDATA = "FinData"
+
+let toDoArr = [];
+let finArr = [];
+
+function loadDATA(){
+    const loadedTODO = localStorage.getItem(TODODATA);
+    const loadedFIN = localStorage.getItem(FINDATA);
+    
+    if (loadedTODO !== null) {
+        const parsedTODO = JSON.parse(loadedTODO); // parse, 가져온 것을 자바스크립트 object로 변환시켜줌
+        parsedTODO.forEach(function(input) {
+            printData(input.text,parseInt(input.id),true)
+        });
+    }
+    if (loadedFIN !== null) {
```

Comment:
```suggestion
    if (loadedFin) {
```
여기도 이렇게 간단히 해볼 수 있겠네요.